model/license_test.go

```diff
@@ -116,6 +116,15 @@ func TestLicenseIsExpired(t *testing.T) {
 	assert.False(t, l1.IsExpired())
 }
 
+func TestLicenseIsPastGracePeriod(t *testing.T) {
+	l1 := License{}
+	l1.ExpiresAt = GetMillis() - LICENSE_GRACE_PERIOD - 1
+	assert.True(t, l1.IsPastGracePeriod())
+
+	l1.ExpiresAt = GetMillis() + 1
```

Comment:
sure, we can change it, the thinking was that if one specifies a value somebody can question it later- like why that specific value, does it have a certain meaning - with "1" it is more clear (to me) that it is just for "going over/under limit" purpose, but I'm 0/5 on this.

Reply:
yes, but 1ms is a very small amount of time and if the cpu switch the context between the two GetMillis, can fail the tests. I think GetMillis() + 1000 is pretty obvuious that you are saying "one second later" instead of "1 millisecond later".

Reply:
agree, didn't think in terms of milliseconds/seconds.